README.md

```diff
@@ -2,8 +2,84 @@
 
 Repository containing pipeline for processing spinal cord injury (SCI) patients (both DCM and tSCI) using the [Spinal Cord Toolbox (SCT)](https://github.com/spinalcordtoolbox/spinalcordtoolbox).
 
-Steps:
+Pipeline steps:
 1. DICOM to nii (BIDS) conversion
 2. Processing (spinal cord and lesion segmentation, vertebral labeling)
 3. Quality control (QC) + manual compression level labeling
 4. Lesion metric computation
+
+
+## Dependencies
+
+[Spinal Cord Toolbox v6.4](https://github.com/spinalcordtoolbox/spinalcordtoolbox/releases/tag/6.4)
+[dcm2niix >= v1.0.20220505](https://github.com/rordenlab/dcm2niix?tab=readme-ov-file#install)
+
+## Installation
```

Comment:
Consider adding numbers before each steps and a TOC?

Reply:
good point! TOC added in https://github.com/sct-pipeline/balgrist-sci/pull/2/commits/62b47507786b194edfce216f35e1ea5a714eb1cb